### ISSUE TITLE
Some things difficult to see on a projector

### ISSUE THREAD

**Issue by bsnouffer** [shcaba/FishPath-updates#318]
Noted during the Feb 2020 training:

List of things that I noticed were difficult to see/read when using the tool on a projector at the Feb 2020 training. Please add to the list if anyone notices other areas that are affected.

fishery information screen, light grey labels
the number in the caveat/criteria icon on the results screen (maybe change to black font)
borders between light tan background and content boxes (looks like all white background)
light grey text in general
button borders (blend in with white background)
question subtext is too small to read from distance (usually have to zoom in during workshop)

**Comment by bsnouffer** [comment]
I made the following updates to try to address this. Unfortunately I do not have a projector to test on. Most of the changes are slight to give more contrast. These are all easy to change if things are still too difficult to see on projectors, but I did not want to make too drastic of changes.

- Changed font for caveats and criteria to black (except for red and black backgrounds), this seemed to give the most contrast and makes the numbers stand out more on a computer screen as well.
- Change the green for caveats and criteria to a slightly lighter green. This helps with the contrast of the black number
- Increased the size of the criteria icons in the data collection section to match the size in the assessment section
- Slightly darkened the background of the entire site
- Slightly darkened the text of the text input boxes in the fishery info screen
- Increased the font size of the question descriptions
